### DISCUSSION TITLE
The site says that my campus is not in my city

### DISCUSSION THREAD

**Discussion by Alexey-BD** [community/community#38296] [GitHub Education]
On the site, when I received the Student Developer Pack, it says that my campus is not in my city... But even by the name of the university, it is clear that it is in my city. I simply cannot confirm that I am a student of this university. What should I do about it?
![image](https://user-images.githubusercontent.com/117467194/200031759-e1d88722-ae69-4a8f-88a8-0d39bb7a3621.png)

**Comment by david-engelmann** [comment]
Are you using a VPN?

**Comment by samus-aran** [comment]
Hi @Alexey-BD 👋 

How did you get on? @david-engelmann is correct to ask if you are using a VPN. It's important for us to know the location of the students in relation to their institute so that we can assess accurately if you are a remote or campus learner. If you are not using the browser location we are unable to do this which may lead to a rejected application.
